.circleci/config.yml

```diff
@@ -260,9 +260,7 @@ jobs:
           name: parallel go race tests
           command: |
             set +e
-            # filter internal/promqltests because it has special go.mod
-            TESTFILES=$(go list ./... | grep -v internal/promqltests | circleci tests split --split-by=timings)
-            echo $TESTFILES
```

Comment:
I like the `echo $TESTFILES` , should we keep it?

Reply:
:+1: will restore it